docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -68,6 +68,8 @@ It is an exact path match because it does not end with `*`.
 <3> This permission set references the previously defined policy.
 `roles1` is an example name; you can call the permission sets whatever you want.
 
+WARNING: The `/forbidden` exact path in the example above will not secure the `/forbidden/` path. Don't forget to add new exact path for the `/forbidden/` path.
```

Comment:
@michalvavrik Is it a breaking change or it it the way it already works now ?

Reply:
That's how it works and I'd like users to beware of it.

Reply:
because I suppose HTTP request to /forbidden/ will go to resource with /forbidden right?

Reply:
>because I suppose HTTP request to /forbidden/ will go to resource with /forbidden right?

I think so (though not 100% sure if it varies between Reactive and Classic), which is why it is a bit unusual that cover both `/forbidden` and `/forbidden/` users have to specify two HTTP security policy paths.

Reply:
agreed, but that has nothing to do with this PR (didn't change by this PR), we should improve it after this one. I'll not forget about it, no worry.